app/views/filters/_form.html.erb

```diff
@@ -52,12 +52,14 @@
       <div id="granular_form" style="<%= f.object.granular? ? '' : 'display:none' %>">
         <%= checkbox_f f, :unlimited?,
                        :label_help => (_("If the unlimited field is enabled, the filter applies to all resources of the selected type. If the unlimited </br> field is disabled, you can specify further filtering using Foreman search syntax in the search field.</br> If the role is associated with %s, the filters are not considered unlimited</br> as they are scoped accordingly.") % org_loc_string('or')).html_safe %>
-
-
-        <%= autocomplete_f f, :search,
-                           :disabled => f.object.unlimited?,
-                           :full_path => search_path(f.object.resource_type),
-                           :control_group_id => 'search_group' %>
+        
+        <%= autocomplete_f f, nil,
+                          :disabled => f.object.unlimited?,
+                          :full_path => search_path(f.object.resource_type),
+                          :control_group_id => 'search_group',
+                          :resource_type => f.object.resource_type,
+                          :search_query => @filter.search
+                          %>
```

Comment:
does this mean the method signature changes ? (so a developer who uses this method needs to change the parameters passed to it?)

Reply:
@ohadlevy The method signature is still the same,
it receives `f, attr, options` though the `attr` doesn't affect anything now, it is just there to keep the signature as it was,
in the options, I kept the `disabled, full_path, control_group-id`
and added `controller` and `search_query` as initial data to update the `Redux store`

please tell me if there is anything else to support, thanks.

Reply:
@Laviro my feedback is based on the fact that the way you call the method changed, so i was wondering if it got broken somehow? would it work if you undo this change? in that case, maybe we won't need to change it all?

Reply:
@ohadlevy well, I added  `search-query` as an option, 
though before it was called inside the helper, from the `f.object`,
so I can implement that too.

As for the `controller` I've added as an option,
it is not required but it helps to indicate in `Redux`,
what is behind the request.

So there shouldn't be braking changes.